packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vdsm/tasks/libvirt_presets_configure.yml

```diff
@@ -0,0 +1,32 @@
+- block:
+  - name: create prestes copy for custom libvirt presets
+    copy:
+      src: /usr/lib/systemd/system-preset/90-default.preset
+      dest: /usr/lib/systemd/system-preset/92-default.preset
+      remote_src: yes
+
+  - name: disable libvirt modular daemon presets
+    lineinfile:
+      path: /usr/lib/systemd/system-preset/92-default.preset
```

Comment:
Is this file created by us? If so, can we use existence of the file as a check to no perform masking on each host deploy run?

Also if would be good if we could add a comment into the config file, that this is oVirt internal configuration and should not be updated manually.

Reply:
yes, the file is created by us (same file line ~3). i hope that i understand you correctly, but even though we use the presets to disable the services of the libvirt multi daemon i think its important to mask them that by accident they wont restart and conflict with the libvirtd daemon(unfortunately presets cant mask services just disable them), so only checking if the file exists and not mask the services will not be enough no? or you think we should drop the masking part?

good idea, will add a  comment into the config file.

Reply:
if it's our file it should have some ovirt name, not "default"

Reply:
the `/usr/lib/systemd/system-preset/90-default.preset` is shipped with the os, we override it with our own prested we need to keep the original name.

Reply:
does it not override everything if you provide a file with the right conf after 92?